.buildkite/pipeline.yml

```
@@ -75,6 +75,10 @@ steps:
           - "--fail-fast"
           - "--app-activity=com.bugsnag.mazeracer.SplashScreenActivity"
           - "--app-package=com.bugsnag.mazeracer"
+      test-collector#v1.10.2:
+        files: "reports/TEST-*.xml"
+        format: "junit"
+        branch: "^master|next$$"
```

Sorry, late breaking comment:
```suggestion
        branch: "^main|next$$"
```